src/pages/dashboardGroupEdit/components/GroupForm.tsx

```diff
@@ -279,12 +279,12 @@ function GroupForm({
   });
 
   useEffect(() => {
-    if (formik.dirty || (productSelected && !isEdit)) {
+    if (formik.dirty || (productSelected && !isEdit && !isClone)) {
```

Comment:
this way maybe each change to the selected product could not brought to register the unload event
We should to test better
But I think that the issue JIRA is wrong... let's await for a confirmation